### ISSUE TITLE
CrewAI agent integrations

### ISSUE THREAD

**Issue by RYangData** [Chainlit/docs#142]
Is there any plan to integrate with CrewAI? Would be good to have this! :D

**Comment by constantinidan** [comment]
We'd love to see a community contribution on this one!

**Comment by dahifi** [comment]
@RYangData My team has done some experimenting integrating CrewAI in Chainlit, but we backed off of it in favor of our own OAI Event Handler. We are still interested in building out similar features if you want to discuss it in the Discord. I know there was someone on there who was asking about this several weeks ago. 

Their assistant class is really solid -- trying to maintain parity between local config and OAI Assistants is challenge right now, but we didn't want to wait for them to roll out their V2 update so we did our own. 

We also never finished overriding their completion call. Everything was made to write to terminal and we didn't have sufficient knowledge at the time to build the updated OAI components and figure out how Crew worked. 

I have also personally built a custom tool wrapper using their base class, but I haven't been keeping up with their development efforts.
